e2e/tests/user/registration.spec.ts

```diff
@@ -6,7 +6,7 @@ const configs = require('../../resources/workbench-config');
 // set timeout globally per suite, not per test.
 jest.setTimeout(2 * 60 * 1000);
 
-describe('User registration tests:', () => {
+describe.skip('User registration tests:', () => {
```

Comment:
Skip because test is broken without a valid contact email. `hermione.owner@quality.firecloud.org` is not usable anymore.
Ticket to trace work https://precisionmedicineinitiative.atlassian.net/browse/RW-4648